src/ClimaLand.jl

```diff
@@ -127,7 +127,9 @@ function initialize_lsm_aux(land::AbstractLandModel, land_coords)
     domains = lsm_aux_domain_names(land)
     additional_aux = map(zip(types, domains)) do (T, domain)
         zero_instance = ClimaCore.RecursiveApply.rzero(T)
-        map(_ -> zero_instance, getproperty(land_coords, domain))
+        f = map(_ -> zero_instance, getproperty(land_coords, domain))
+        fill!(f, zero_instance)
```

Comment:
```suggestion
        fill!(Fields.field_values(f), zero(eltype(f)))
```
This way we know that this is mask-unaware